distsql/select_result.go

```diff
@@ -160,6 +160,9 @@ type selectResult struct {
 
 func (r *selectResult) fetchResp(ctx context.Context) error {
 	defer func() {
+		if enableTelemetry, telemetryErr := telemetry.IsTelemetryEnabled(r.ctx); telemetryErr != nil || !enableTelemetry {
```

Comment:
It is not to fix. it just skips this problem. because the telemetry is in deprecated.